src/templates/landing_pages/ico.html.tt

```diff
@@ -165,7 +165,21 @@
                     </p>
                 </div>
             </div>
-            <hr>
+            <div id="lykke-section" class="content-section center-text fill-bg-color">
+                <div class="container">
+                    <div class="gr-row">
+                        <div class="gr-10 gr-push-1">
+                            <h2>[% l('Trade your tokens on the Lykke Exchange') %]</h2>
+                            <img src="[% request.url_for('images/ico/icons/lykke_logo.svg') %]"/>
+                            <p>[% l('We\'ve reached an agreement to list the tokens from our forthcoming ICO on the <a href="[_1]" target="_blank" rel="noreferrer noopener">Lykke Exchange</a>. Lykke is a fintech company that\'s building a global marketplace on the blockchain.', 'https://www.lykke.com/') %]</p>
```

Comment:
Remove the sentence "Lykke is a fintech company that\'s building a global marketplace on the blockchain." - it's unnecessary.